.github/workflows/ci.yaml

```diff
@@ -10,10 +10,10 @@ jobs:
       fail-fast: true
       matrix:
         os: ["ubuntu-latest", "macos-latest"]
-        python-version: ["3.9", "3.10", "3.11"]
+        python-version: ["3.9", "3.11", "3.12"]
```

Comment:
Staying on 3.9?

Reply:
Yep, it's not needed anymore, see here: https://scientific-python.org/specs/spec-0000/#support-window